src/operator/gitea.ts

```diff
@@ -1,112 +1,149 @@
-import Operator from '@dot-i/k8s-operator'
+import Operator, { ResourceEventType } from '@dot-i/k8s-operator'
 import * as k8s from '@kubernetes/client-node'
-import { KubeConfig } from '@kubernetes/client-node'
 import stream from 'stream'
 
+import {
+  CreateHookOption,
+  CreateOrgOption,
+  CreateRepoOption,
+  CreateTeamOption,
+  EditRepoOption,
+  OrganizationApi,
+  Repository,
+  RepositoryApi,
+  Team,
+} from '@redkubes/gitea-client-node'
+import { keys } from 'lodash'
+import { doApiCall } from '../utils'
+import { GITEA_OPERATOR_NAMESPACE, GITEA_URL, GITEA_URL_PORT, cleanEnv } from '../validators'
+import { orgName, otomiChartsRepoName, otomiValuesRepoName, teamNameViewer, username } from './common'
+
+// Interfaces
+interface hookInfo {
+  id?: number
+  hasHook: boolean
+}
+
 interface groupMapping {
   [key: string]: {
     otomi: string[]
   }
 }
 
-const kc = new KubeConfig()
+interface DependencyState {
+  giteaPassword: string | null
+  teamConfig: any
+  oidcClientId: string | null
+  oidcClientSecret: string | null
+  oidcEndpoint: string | null
+  teamNames: string[] | null
+}
+
+// Constants
+const localEnv = cleanEnv({
+  GITEA_URL,
+  GITEA_URL_PORT,
+  GITEA_OPERATOR_NAMESPACE,
+})
+
+const giteaUrl = `${localEnv.GITEA_URL}:${localEnv.GITEA_URL_PORT}`
+const giteaOperatorNamespace = localEnv.GITEA_OPERATOR_NAMESPACE
```

Comment:
I would follow the conventions for a constant here, now it looks like its a variable in the code
```suggestion
const GITEA_APP_OPERATOR_NAMESPACE = localEnv.GITEA_OPERATOR_NAMESPACE
```

Reply:
Updated!